test/chpldoc/include/test-DocsInclude.doc.chpl

```diff
@@ -0,0 +1 @@
+import DocsInclude;
```

Comment:
Is the reason for this file simply to dodge the fact that `DocsInclude.notest` doesn't just squash the recursion of the test system into `DocsInclude`, but also inadvertently squashes the DocsInclude.chpl file itself?  Would moving a NOTEST into the subdirectory accomplish the same thing and remove the need for the extra files here?

While not a strong case to worry about, this "`.notest` doing double duty" case adds to my reluctance that naming the submodule subdirectory after the module name itself may be more fragile / confusing than helpful.

(Not directly related to this, but since we're on the topic: I've been wondering whether we could/should permit submodules to live in multiple potential places.  For example, if `MyMod/MySubMod.chpl` isn't found, maybe `./MySubMod.chpl` might also be considered as a candidate for resolving the `include`.

Reply:
> Is the reason for this file simply to dodge the fact that DocsInclude.notest doesn't just squash the recursion of the test system into DocsInclude, but also inadvertently squashes the DocsInclude.chpl file itself? 

No. The problem is that DocsInclude.doc.chpl would only be allowed to be in the module `DocsInclude.doc` which isn't a legal module name (at least not a long one). I might have been more conservative than necessary (it might be that if there is no `module` declaration and it's implicit, this would work out OK).

> While not a strong case to worry about, this ".notest doing double duty" case adds to my reluctance that naming the submodule subdirectory after the module name itself may be more fragile / confusing than helpful.

Huh, I thought it was nice...

> Not directly related to this, but since we're on the topic: I've been wondering whether we could/should permit submodules to live in multiple potential places. 

I think the 1st try should only look in the one place.